app/src/components/googleMap.tsx

```diff
@@ -359,22 +389,22 @@ class SimpleMap extends React.Component<MapProp, MapState> {
                             .getMarkers()
                             .map((m: GoogleMarkerIntf) => m.destination)
                             .sort(sortDestinationsDesc); // make sure they're sored. who knows whether google API keep markers sorted
-
+                        
                         const sameCity = false;
                         //const sameCity = destsInCluster
                         //    .map((d: IDestination) => d.cityName)
                         //    .filter((value: string, index: number, self: string[]) => self.indexOf(value) === index)
                         //    .length === 1;
-
+                        
                         const topMarker = destsInCluster[0];
-
+                        
                         return (
                             <PriceTagMarker
                                 key={idx}
                                 lat={topMarker.lat}
                                 lng={topMarker.lng}
-                                //lat={cluster.getCenter().lat()} // to display it in the middle of cluster
-                                //lng={cluster.getCenter().lng()} // (onMouseEnter should also be updated then)
+                                // to display it in the middle of cluster
+                                // (onMouseEnter should also be updated then)
```

Comment:
don't remove this comments

Reply:
I think this was removed by WebStrom, and I will add it back